R/html_document.R

```diff
@@ -673,8 +683,13 @@ html_highlighters <- function() {
   c(highlighters(), "textmate")
 }
 
-default_mathjax <- function() {
-  paste0("https://mathjax.rstudio.com/latest/", mathjax_config())
```

Comment:
Wow, I am surprised. Does CRAN allow usage of `:::`?
Anyway, I will keep the function.
I would recommend export the function to avoid such a spurious removal.